.circleci/config.yml

```diff
@@ -296,19 +296,12 @@ defaults:
 
   - base_osx: &base_osx
       macos:
-        xcode: "13.2.0"
+        xcode: "14.2.0"
+      resource_class: macos.x86.medium.gen2
       environment:
         TERM: xterm
         MAKEFLAGS: -j5
 
-  - base_osx_large: &base_osx_large
-      macos:
-        xcode: "13.2.0"
-      resource_class: large
```

Comment:
Looks like there's no replacement for `large` but in that case I think we should keep using it as long as possible. Switching to `medium` will slow down our `b_osx` job and macOS compilation and testing is already much slower than we would like.

I see this on https://circleci.com/docs/using-macos/:

> The `large` resource class is only available for customers with an annual contract.

Also, looking at our PRs, `b_osx` still executes with 8 cores and 16 GB RAM so apparently we do count as "customers with an annual contract". In that case let's stay on it and only convert `medium` to `macos.x86.medium.gen2`.

Reply:
Exactly. I noticed that as well, and thought that we could try the `m1`, but indeed it is better to keep `base_osx_large` as it is until we fix the other issue to produce ARM releases.